it/controllers/LookupPageISpec.scala

```diff
@@ -169,6 +171,10 @@ class LookupPageISpec extends IntegrationSpecBase {
           doc.title shouldBe lookupPage.title.get + " - NAV_TITLE - GOV.UK"
           doc.h1.text() shouldBe lookupPage.heading.get
 
+          doc.getElementsByClass("afterHeadingText") should have(
```

Comment:
Should that be getElementById?

Reply:
Yes it should - that would be intellij trying to be smart on me.

Reply:
There's a couple of others in this file I think